submission/persistence/src/main/kotlin/ac/uk/ebi/biostd/persistence/service/FilePersistenceService.kt

```diff
@@ -21,23 +23,23 @@ class FilePersistenceService(
     }
 
     private fun persistFiles(submission: ExtSubmission) {
-        copy(submission.allFiles, submission.relPath)
-        copy(submission.allReferencedFiles, submission.relPath)
-    }
+        val submissionFolder = getSubmissionFolder(submission.relPath)
+        val temporally = createTempFolder(submissionFolder, submission.accNo)
 
-    private fun generateOutputFiles(submission: ExtSubmission) {
-        generateOutputFiles(submission.toSimpleSubmission(), submission, submission.accNo)
-        submission.allFileListSections.forEach { generateOutputFiles(it, submission, it.fileName) }
+        submission.allFiles.forEach { it.file.renameTo(temporally.resolve(it.fileName)) }
```

Comment:
is needed because of submission folder is a source so we can not delete these files (as one may be needed). About move we can not use it as internal implementation for that one is copying.